src/transactions/transaction_builder/transaction_builder.ts

```diff
@@ -126,6 +131,111 @@ export function generateTransactionPayload(args: GenerateTransactionPayloadData)
   );
 }
 
+export async function generateTransactionPayloadWithRemoteABI(
+  args: EntryFunctionDataWithABI,
+  aptosConfig: AptosConfig,
+): Promise<TransactionPayloadEntryFunction>;
+export async function generateTransactionPayloadWithRemoteABI(
+  args: MultiSigDataWithABI,
+  aptosConfig: AptosConfig,
+): Promise<TransactionPayloadMultisig>;
+export async function generateTransactionPayloadWithRemoteABI(
+  args: GenerateTransactionPayloadDataWithABI,
+  aptosConfig: AptosConfig,
+): Promise<TransactionPayload>;
+
+export async function generateTransactionPayloadWithRemoteABI(
```

Comment:
On going discussion on if this should be named generateTransaction to make it clearly the default for sending transactions

Reply:
Fixed